x-pack/plugins/enterprise_search/public/applications/app_search/components/search_ui/components/search_ui_form.test.tsx

```diff
@@ -43,10 +46,6 @@ describe('SearchUIForm', () => {
     setMockActions(actions);
   });
 
-  beforeEach(() => {
-    jest.clearAllMocks();
-  });
-
```

Comment:
I'm kinda medium on this change - I probably would have just left line 186 as a beforeEach instead of a beforeAll 🤷 nbd either way though, def not a blocker

Reply:
Yeah, I'd be fine either way as well.